src/odemis/dataio/tiff.py

```diff
@@ -804,19 +807,13 @@ def _updateMDFromOME(root, das):
                 continue # might be a thumbnail, it's alright
             da.metadata.update(mdp)
 
-        # Convert the DeltaT's into PIXEL_DUR + TIME_OFFSET
+        # Update metadata of each da, so that they will be merged
```

Comment:
All this block needs to be only running if deltats is not empty (ie, `if deltats:`)